contrib/qubes/README.md

```diff
@@ -0,0 +1,243 @@
+# qubes-fwupd
+
+fwupd wrapper for QubesOS
+
+## Table of Contents
+
+* [Requirements](#Requirements)
+* [Usage](#Usage)
+* [Installation](#Installation)
+* [Testing](#Testing)
+* [Whonix support](doc/whonix.md)
+* [UEFI capsule update](doc/uefi_capsule_update.md)
+* [Heads update](doc/heads_update.md)
+
+## OS Requirements
+
+**Operating System:** Qubes OS R4.1 or Qubes OS R4.0
+
+**Admin VM (dom0):** Fedora 25 or higher
+
+**Template VM:** Fedora 30
+
+**Whonix VM:** whonix-gw-15
+
+**fwupd version - dom0:** 0.9.5 or higher
+
+**fwupd version - VMs:** 1.2.6 or higher
+
+qubes-fwupd does not support dom0 updates and downgrades for fwupd 0.9.5 and
+older. Use sys-usb to update external devices.
```

Comment:
So everything is Fedora based?  If that's the case - let's not worry about any of the Debian packaging stuff.

Reply:
Oh Whonix is Debian based?  Do the changes get pushed up to Debian?

Reply:
`whonix-gw-15` VM is based on Debian buster